atom/browser/api/atom_api_web_contents.cc

```diff
@@ -754,6 +754,11 @@ content::JavaScriptDialogManager* WebContents::GetJavaScriptDialogManager(
   return dialog_manager_.get();
 }
 
+void WebContents::OnAudioStateChanged(content::WebContents* web_contents,
+                                      bool audible) {
+  Emit("audio-state-changed", audible);
```

Comment:
If this is public API it should be documented, if it is private API we typically prefix with a `-` so this would emit `-audio-state-changed`

Reply:
I prefer keeping it private, doesn't make sense for both the event and the api to be present. Deciding to remove one over the other can be done in a separate PR. Have updated the PR to reflect the event name, thanks!